SharpBucket/SharpBucket.cs

```diff
@@ -245,6 +245,32 @@ private async Task<T> SendAsync<T>(object body, Method method, string overrideUr
             return await authenticator.GetResponseAsync<T>(overrideUrl, method, body, requestParameters, token);
         }
 
+        private Uri SendAndGetRedirectLocation(object body, Method method, string overrideUrl, IDictionary<string, object> requestParameters)
```

Comment:
The name of the method is really strange here
I tried to follow the pattern of the existing Send methods, but I not found of the result.

Maybe we should just name the method GetRedirectLocation ?

Maybe providing a body and Method is totally overkill and we should just delete that layer ?
Even if it's HTTP compliant to get a redirect response for all verbs (POST, PUT, ...) and not just GET, I'm not convinced that BitBucket will exploit this, and I'm even less convinced that we may want to just retrieve the redirect URL of a POST, PUT or DELETE operation. 
GET is used to retrieve information, and a redirect URL could be the information we try to GET.
But if I do a POST, PUT or DELETE, I'm not trying to get some information, so why could we want to functionally retrieve the redirect location of such requests ?

@MitjaBezensek are you agree with my reasoning ?
In that case I will just expose the ability to "GetRedirectLocation" for GET methods, and drop the body and method parameters on all the chain

Reply:
I've follow my reasoning and simplify the signature to support only GET verb